Documentation/ceph-object-bucket-claim-crd.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Object Bucket Claim CRD
+weight: 2800
+indent: true
+---
+
+# Ceph Object Bucket Claim CRD
+
+Rook allows creation and customization of object bucket claim through the custom resource definitions (CRDs). The following settings are available for Ceph Object Bucket Claim. It also requires storage class
+
+## Sample
+##### Stoarge class
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+   name: rook-ceph-bucket
+provisioner: ceph.rook.io/bucket
+# set the reclaim policy to delete the bucket and all objects
+# when its OBC is deleted.
```

Comment:
Spell out "object bucket claim" here because "OBC" has not yet been defined for the reader

Reply:
if above changes are accepted then obc will be defined above...

Reply:
Removed comments in yaml section